os/arch/arm/src/armv7-a/arm_blocktask.c

```diff
@@ -130,6 +130,7 @@ void up_block_task(struct tcb_s *tcb, tstate_t task_state)
            */
 
           rtcb = this_task();
+	  up_restoretask(rtcb);
```

Comment:
Please the comments for calling up_restoretask. `/* Restore rtcb data for context switching */`